src/main/java/com/matera/cadastrocentral/telephone/TelephoneController.java

```diff
@@ -46,20 +42,13 @@ public Telephone insertTelephone(@PathVariable("clientId") UUID clientId,
     }
 
     // 4. Delete a specific telephone
-    @DeleteMapping("/{telephoneId}/telephone")
-    public void deleteTelephone(@PathVariable("telephoneId") UUID telephoneId) {
-        telephoneService.deleteTelephone(telephoneId);
+    @DeleteMapping("clients/{clientId}/telephones/{telephoneId}")
+    public void deleteTelephone(@PathVariable("clientId") UUID clientId, @PathVariable("telephoneId") UUID telephoneId) {
+        telephoneService.deleteTelephone(clientId, telephoneId);
     }
 
-    // 5. Find specific telephone for specific client
-    @GetMapping("/client/{clientId}/telephone/{telephoneId}")
-    public Optional<Telephone> getByClientIdAndTelephoneId(@PathVariable("clientId") UUID clientId,
-                                                           @PathVariable("telephoneId") UUID telephoneId) {
-        return telephoneService.findByClientIdAndTelephoneId(clientId, telephoneId);
-    }
-
-    // 6. Update telephone in database
-    @PutMapping("/telephone")
+    // 5. Update telephone in database
+    @PutMapping("/clients/{clientId}/telephones")
```

Comment:
Here is missing the {telephoneId} in the path, isn't? We are going to alter a specific telephone from a specific client.

Reply:
good catch @gabrieloswaldo, yeah, it's missing it

Reply:
I could add it in the path, but the method already recieves a TelephoneDTO with the clientId, so it thought It would be redundant. But I will add the {telephoneId} path variable to keep the consistancy. thanks!

Reply:
we should always use the ids from the path (it's REST  standards), unless we want to change the ids somehow, thats not the case here :)